src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1422,4 +1422,13 @@
   <data name="DoNotUseOutAttributeStringPInvokeParametersTitle" xml:space="preserve">
     <value>Do not use 'OutAttribute' on string parameters for P/Invokes</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="AvoidAssemblyLocationInSingleFileDescription" xml:space="preserve">
+    <value>Accessing Assembly location or file path is not valid when publishing as a single-file.</value>
+  </data>
+  <data name="AvoidAssemblyLocationInSingleFileTitle" xml:space="preserve">
+    <value>Avoid '{0}'`' when publishing as a single-file</value>
```

Comment:
I don't know if I did a mistake on my suggestion or what happened but there is a problem with the title.

```suggestion
    <value>Avoid '{0}' when publishing as a single-file</value>
```